freqtrade/optimize/backtesting.py

```diff
@@ -252,8 +251,17 @@ def _get_sell_trade_entry(self, trade: LocalTrade, sell_row: Tuple) -> Optional[
         sell = self.strategy.should_sell(trade, sell_row[OPEN_IDX],  # type: ignore
                                          sell_row[DATE_IDX], sell_row[BUY_IDX], sell_row[SELL_IDX],
                                          low=sell_row[LOW_IDX], high=sell_row[HIGH_IDX])
-        if sell.sell_flag:
 
+        time_in_force = self.strategy.order_time_in_force['sell']
+        # confirm_trade_exit
+        if not strategy_safe_wrapper(self.strategy.confirm_trade_exit, default_retval=True)(
+                pair=trade.pair, trade=trade, order_type='limit', amount=trade.amount,
+                rate=sell_row[LOW_IDX],
```

Comment:
I don't think you can (or should) use `sell_row[LOW_IDX]` here. "rate" should correspond to the rate used - which low does only do in some cases.

I'd rather:

* Move all of this into the below if block (so it's only executed if a sell-signal would actually happen)
* use the calculated `closerate` as rate (this is the rate a order would be placed)

This would make behaviour aligned to how it works in trade mode, where this is only called once a sell is triggered and allows you to stop the signal "last-second" once the sell-price (and everything else) is already predefined.


Apart from this, i think this looks good. (note that i did push one commit to fix the "isort" error.

Reply:
done